_data/redirects.yaml

```
@@ -100,4 +100,8 @@
 - source: /egm/agenda
   destination: /assets/agendas/agenda_22_10_27.pdf
 - source: /egm
-  destination: /assets/agendas/agenda_22_10_27.pdf
\ No newline at end of file
+  destination: /assets/agendas/agenda_22_10_27.pdf
+  
+ # Escape Room
+ - source: /escape
+  destination: https://www.guildofstudents.com/events/6531/6953/
```

```suggestion
 - source: /escape
   destination: https://www.guildofstudents.com/events/6531/6953/
```
literally broken smh